lib/TaxId.php

```diff
@@ -14,11 +14,11 @@
  * @property string $object String representing the object's type. Objects of the same type share the same value.
  * @property null|string $country Two-letter ISO code representing the country of the tax ID.
  * @property int $created Time at which the object was created. Measured in seconds since the Unix epoch.
- * @property string|\Stripe\Customer $customer ID of the customer.
+ * @property null|string|\Stripe\Customer $customer ID of the customer.
  * @property bool $livemode Has the value <code>true</code> if the object exists in live mode or the value <code>false</code> if the object exists in test mode.
  * @property string $type Type of the tax ID, one of <code>ae_trn</code>, <code>au_abn</code>, <code>br_cnpj</code>, <code>br_cpf</code>, <code>ca_bn</code>, <code>ca_qst</code>, <code>ch_vat</code>, <code>cl_tin</code>, <code>es_cif</code>, <code>eu_vat</code>, <code>hk_br</code>, <code>id_npwp</code>, <code>in_gst</code>, <code>jp_cn</code>, <code>kr_brn</code>, <code>li_uid</code>, <code>mx_rfc</code>, <code>my_frp</code>, <code>my_itn</code>, <code>my_sst</code>, <code>no_vat</code>, <code>nz_gst</code>, <code>ru_inn</code>, <code>sa_vat</code>, <code>sg_gst</code>, <code>sg_uen</code>, <code>th_vat</code>, <code>tw_vat</code>, <code>us_ein</code>, or <code>za_vat</code>. Note that some legacy tax IDs have type <code>unknown</code>
  * @property string $value Value of the tax ID.
- * @property \Stripe\StripeObject $verification
+ * @property null|\Stripe\StripeObject $verification Tax ID verification information.
```

Comment:
those two are expected because the resource will be re-used with `Account` in the future and so `customer` and `verification` are now optional